CesiumGltfReader/src/GltfReader.cpp

```diff
@@ -357,8 +361,106 @@ ModelReaderResult GltfReader::readModel(
   return result;
 }
 
+/*static*/
+Future<ModelReaderResult> GltfReader::resolveExternalData(
+    AsyncSystem asyncSystem,
+    const std::string& baseUrl,
+    const HttpHeaders& headers,
+    std::shared_ptr<IAssetAccessor> pAssetAccessor,
+    ModelReaderResult&& result) {
+  // TODO: Can we avoid this copy conversion?
+  std::vector<IAssetAccessor::THeader> tHeaders(headers.begin(), headers.end());
+
+  if (!result.model) {
+    return asyncSystem.createResolvedFuture(std::move(result));
+  }
+
+  size_t externalBufferCount = 0;
+  for (const Buffer& buffer : result.model->buffers) {
+    if (buffer.uri) {
```

Comment:
I think we should explicitly exclude data URIs. With default settings, they'll already have been loaded and cleared before we get here. But if the user disabled the "clear data URIs" option, we'll end up loading them twice.